build_debian.sh

```diff
@@ -28,6 +28,10 @@
 ## Enable debug output for script
 set -x -e
 
+CONFIGURED_ARCH=$([ -f .arch ] && cat .arch || echo $HOST_ARCH)
+[[ $SONIC_ARCH == "" ]] && SONIC_ARCH=$CONFIGURED_ARCH
```

Comment:
>SONIC_ARCH [](start = 27, length = 10)

what is the diff between SONIC_ARCH and $CONFIGURED_ARCH, can we keep one ARCH?

Reply:
ok, will do this change

Reply:
Done